index.html

```diff
@@ -39,77 +39,312 @@
       }
     </style>
   </head>
-  <body>
-    <section class="main">
-      <div>
-        <!-- navbar setup -->
-        <header>
-          <nav>
-            <ul class="menu">
-              <li>Portfolio</li>
-              <li>About</li>
-              <li>Contact</li>
-            </ul>
-            <div class="h-menu">
-              <a href="">Welcome</a>
-              <div>
-                <div class="bar"></div>
-                <div class="bar"></div>
-                <div class="bar"></div>
-              </div>
-            </div>
-          </nav>
-        </header>
-        <!-- main body -->
-        <main class="body">
+  <body class="main">
+    <!-- navbar setup -->
+    <header>
+      <nav>
+        <ul class="menu">
+          <li>Portfolio</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <div class="h-menu">
+          <a href="">Welcome</a>
           <div>
-            <section>
+            <div class="bar"></div>
+            <div class="bar"></div>
+            <div class="bar"></div>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <!-- main body -->
+    <main class="body">
+      <section>
+         <br />
+          <h1>Hi, I'm Dagmawi</h1>
+          <h2>I’m a Software Developer</h2>
+          <br />
+      </section>
+      <section>
+        <p>
+          My name is Dagmawi Yohannes I'm a Front End Engineer based in
+          Addis Ababa, Ethiopia ☀️. I describe myself as a passionate
+          developer who loves coding, open source, and the web platform
+        </p>
+        <br />
+      </section> 
+      <div class="h2"></div>
+      <div class="social-links">
+        <ul class="ulist">
+          <li class="ulist-icon">
+            <a href="https://twitter.com/dag_mawi111 "
+              ><img src="./images/twitter.svg" alt="twitter icon"
+            /></a>
+          </li>
+          <li class="ulist-icon">
+            <a href="https://linkedin.com/in/dagmawi-yohannes-b36488236"
+                ><img src="./images/Linkedinicon.svg" alt="linkedin icon"
+            /></a>
+          </li>
+          <li class="ulist-icon">
+            <a href="https://medium.com/dagmawi.yoha03"
+                ><img src="./images/medium.svg" alt="medium icon"
+            /></a>
+          </li>
+          <li class="ulist-icon">
+            <a href="https://github.com/dagmawiii03"
+                  ><img src="./images/github.svg" alt="github icon"
+            /></a>
+          </li>
+        </ul>
+      </section>
+      <section class="pointer">
+        <ul>
+          <li>
+            <a href=""><i class="fas fa-chevron-down white-color"></i></a>
+          </li>
+        </ul>
+      </section>
+      <!--landing pages-->
+      <section class="pages">
+        <section class="header">
+          <h1>
+            My Recent <br />
+            Works
+          </h1>
```

Comment:
- [ ] Since you have already used `h1` for `Hi, I'm Dagmawi` please consider using other heading tags like `h2`, `h3`, `h4`  .... :pray:

Reply:
Done